azure_pipelins/verify_msalcpp_per_pr_ios.yml

```diff
@@ -0,0 +1,61 @@
+# Starter pipeline
+# Start with a minimal pipeline that you can customize to build and deploy your code.
+# Add steps that build, run tests, deploy, and more:
+# https://aka.ms/yaml
+
+trigger: none # Non-PR branches
+pr:
+  branches:
+    include:
+    - '*'
+
+pool:
+  vmImage: 'macOS-10.15'
+
+resources:
+ repositories:
+   - repository: microsoft-authentication-library-for-cpp
+     type: github
+     endpoint: 'GitHub for AzureAD and Azure-Samples (as aadidgit service)'
+     name: AzureAD/microsoft-authentication-library-for-cpp
+     ref: develop
+
+variables:
+  runCodesignValidationInjection: false
```

Comment:
Is this to address the known issue that we have with CI/CD not recognizing MacOS profile? current workaround is to simply change it on XCODE to "sign in to run locally"

Reply:
Hi Peter, sorry for the late replying, this tag is not related to iOS specific